app/controllers/people_controller.rb

```diff
@@ -0,0 +1,27 @@
+class PeopleController < ApplicationController
+
+  def index
+    @all_people = Person.all
+
+    render :index
+  end
+
+  def show
+    @person = Person.find(params[:id])
+
+    @person_id =        @person[:id]
+    @person_name =      @person[:name]
+    @person_fav_genre = @person[:fav_genre]
+
+    render :person
+  end
+
+  def all_tasks
+    @person =      Person.find(params[:id])
```

Comment:
You could just use `@person` as the sole instance variable since it contains all of the individual pieces that you need within the view.